cli/tauri.js/src/runner.ts

```diff
@@ -67,6 +68,18 @@ class Runner {
       ls.stderr?.pipe(process.stderr)
       ls.stdout?.pipe(process.stdout)
       this.beforeDevProcess = ls
+
+      let devTryCount = 0
+      const devTryTimeout = 3000
+      while (!(await isReachable(devPath))) {
+        log('Waiting for your dev server to start...')
+        await new Promise(resolve => setTimeout(resolve, devTryTimeout))
```

Comment:
should we use `setInterval` instead? sometimes devservers take forever to spin up...

Reply:
more than 15s?

Reply:
if you are running 10 electron apps and 100 chrome tabs with only 8gb you might swap and i've had to wait 60 seconds sometimes

Reply:
let's just run it forever then?